python/ql/test/experimental/dataflow/TestUtil/NormalDataflowTest.qll

```diff
@@ -31,3 +31,6 @@ query predicate missingAnnotationOnSINK(Location location, string error, string
     )
   )
 }
+
+/** DEPRECATED: Alias for missingAnnotationOnSink */
+deprecated predicate missingAnnotationOnSINK = missingAnnotationOnSink/3;
```

Comment:
This is just a test-file, so we can just do the rename without deprecation :blush: 

```suggestion
```

Reply:
I'll delete the deprecation with my deprecation-deletion-script in 14+ months anyway.  
And I don't want to run the tests again, and bump the submodule pointer again, so I'll let this one stay.